src/runtime/components/CldOgImage.vue

```diff
@@ -125,7 +117,10 @@ const { url: twitterImageUrl } = useCldImageUrl({
 } as ConstructUrlProps)
 
 const computedTwitterTitle = computed(
-  () => props.twitterTitle || (currentRoute.value.meta?.title as string) || ' ',
+  () =>
+    props.twitterTitle
+    || (currentRoute.value.meta?.title as string)
+    || 'nuxt-cloudinary-og-image',
```

Comment:
thoughts on something like "Learn More" instead? it's overly generic, but at least it's potentially somewhat useful if they dont  set it